src/main/scala/io/iohk/ethereum/network/p2p/Message.scala

```diff
@@ -17,6 +17,8 @@ trait MessageSerializable extends Message {
 
   def toBytes: Array[Byte]
 
+  def underlyingMsg: Message
```

Comment:
In that case it won't be needed, this was added as the interaction with the `Network` layer is done through messages that are serializable (https://github.com/input-output-hk/etc-client/blob/10e5a10226e657d32cf681fbac02d9afb77590a3/src/main/scala/io/iohk/ethereum/network/PeerActor.scala#L253). What do you think the interaction being that way? This was done to take advantage of the `MessageSerializable` so that no implicits need to be passed to the Network layer for serialization.